scripts/clean-package.js

```
@@ -0,0 +1,25 @@
+const fs = require("fs");
+const path = require("path");
+
+// Read the source package.json
+const packagePath = path.join(__dirname, "..", "package.json");
+const distPackagePath = path.join(__dirname, "..", "dist", "package.json");
+
+// Read and parse the package.json
+const packageJson = JSON.parse(fs.readFileSync(packagePath, "utf8"));
+
+// Remove devDependencies
+delete packageJson.devDependencies;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)